gdpr/anonymizers.py

```diff
@@ -13,17 +13,19 @@
 
 
 class FieldAnonymizer:
-    """
-    Field anonymizer's purpose is to anonymize model field accoding to defined rule.
+    """Field anonymizer's purpose is to anonymize model field accoding to defined rule.
```

Comment:
proč tohle děláš?

Reply:
když už to děláš tak vezmi ty ukončovací "" a dej je na konec řádku, bez toho newlinu. Pokud se ti to nevejde na řádek tak z toho udělej multiline dostring (tedy jak to bylo)

Reply:
Upraveno.